lib/build_info.ml

```diff
@@ -17,12 +17,7 @@ let experimental_variant s =
       is_prefix ~affix:Variant.lower_bound_label s.label
       || is_prefix ~affix:Variant.opam_label s.label)
   then true
-  else
-    match s.variant with
-    | None -> false
-    | Some v ->
-        Astring.String.equal "freebsd" (Variant.distro v)
-        || Astring.String.equal "macos-homebrew" (Variant.distro v)
+  else false
```

Comment:
You could remove the conditional if you felt inclined, and just use the disjunction, as on lines 29 and 30, but I could also see an argument for leaving space for else conditions, and it's not a biggie either way.

Reply:
I have removed it - my original thinking was it would have been nice to leave the logic in place to help a future person add a new experimental platform, but they can just check the old commits/git blame.